tasks/styles.js

```diff
@@ -33,13 +38,13 @@ module.exports = (gulp, $, pkg) => {
           console: true
         }]
       }))
-      .pipe($.concat(pkg.title.toLowerCase().replace(/[^a-z]/g,'') + '.scss'))
+      .pipe($.if(options.concat, $.concat(pkg.title.toLowerCase().replace(/[^a-z]/g,'') + '.scss')))
       .pipe($.if(options.sourcemaps, $.sourcemaps.init()))
       .pipe($.sass({
         importer: $.sassModuleImporter(),
         outputStyle: options.outputStyle
       }).on('error', reportError))
-      .pipe($.autoprefixer(pkg.browserslist))
+      .pipe($.autoprefixer())
```

Comment:
Unintended change, remnant from #1 ?

Reply:
No, it is actually intended as the default is looking already looking for the `browserslist` key in the `package.json` file. So passing `pkg.browserslist` was not necessary and can be removed.